splunk_add_on_ucc_framework/install_python_libraries.py

```diff
@@ -116,10 +160,12 @@ def install_python_libraries(
             installer=python_binary_name,
             target=ucc_lib_target,
             libname="splunktaucclib",
+            version="6.4",
+            allow_higher_version=True,
         ):
             raise SplunktaucclibNotFound(
-                f"splunktaucclib is not found in {path_to_requirements_file}. "
-                f"Please add it there because this add-on has UI."
+                f"splunktaucclib is not found in {path_to_requirements_file} or has invalid version. "
+                f"Please add it there and check if it is at least version 6.4, because this add-on has UI."
```

Comment:
I would suggest to separate this into 2 different exceptions / messages:

* there is no `splunktaucclib` found but this add-ons has UI
* `splunktaucclib` found but the developer needs to update it to the version we specify because of <link>

I think it would be easier to understand why build does not actually work.